adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java

```diff
@@ -112,7 +114,7 @@ static MsalOAuth2TokenCache getMsalOAuth2TokenCache(@NonNull final Context appCo
                 new SharedPreferencesFileManager(
                         appContext,
                         DEFAULT_ACCOUNT_CREDENTIAL_SHARED_PREFERENCES,
-                        new StorageHelper(appContext)
+                        new AndroidSdkStorageEncryptionManager(appContext, null)
```

Comment:
should this be using the singleton factory?

Reply:
+1 - a factory would be nice

Reply:
That would be a bigger change in ADAL. Prefer not to do it now.